client/rest-high-level/src/main/java/org/elasticsearch/client/analytics/StringStatsAggregationBuilder.java

```diff
@@ -103,7 +103,7 @@ public BucketCardinality bucketCardinality() {
 
     @Override
     protected ValuesSourceAggregatorFactory innerBuild(AggregationContext context, ValuesSourceConfig config,
-            AggregatorFactory parent, Builder subFactoriesBuilder) throws IOException {
+                                                       AggregatorFactory parent, Builder subFactoriesBuilder) throws IOException {
```

Comment:
Nit: This formatting-only change doesn't change to the correct style.

Reply:
Will fix it with the conflicts! Sorry, completely missed it when I was reviewing the files I changed.